src/join-9.sol

```diff
@@ -35,65 +33,84 @@ interface GemLike {
 
 // For a token that has a fee (PAXG)
 
-contract GemJoin9 is LibNote {
+contract GemJoin9 {
     // --- Auth ---
     mapping (address => uint256) public wards;
-    function rely(address usr) external note auth { wards[usr] = 1; }
-    function deny(address usr) external note auth { wards[usr] = 0; }
+    function rely(address usr) external auth {
```

Comment:
I'd advise to move to the approach of grouping storage, constants/immutables, modifiers, admin functions, user functions. It is much clearer.

Reply:
I've laid out contract elements following the data, events, modifiers, functions order in 5f3a1ab.